tests/ui/test_install.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+
+from pages.desktop.details import Details
+
+
+@pytest.mark.nondestructive
+def test_addon_install(
+        base_url, selenium, firefox, notifications):
+    selenium.get('{}/firefox/addon/ui-test-install'.format(base_url))
+    addon = Details(selenium, base_url)
+    assert 'Ui-Addon-Install' in addon.description_header.name
+    addon.description_header.install_button.click()
+    firefox.browser.wait_for_notification(
+        notifications.AddOnInstallBlocked).allow()
+    firefox.browser.wait_for_notification(
+        notifications.AddOnInstallConfirmation).install()
```

Comment:
Doesn't seem to have any further assertions yet, so still wip?

Reply:
So...there isn't anything else to check as the page doesn't change showing that the addon has been installed. These actions could be a form of a 'assert' as they must complete for the test to pass.

Reply:
Okay, so if `notifications` are the in-browser popup windows, is `AddOnInstallConfirmation` the final "Yes I want to install" pop-up or already the "Installed successfully!" stage? Might need some comments to explain which is which so that we understand what they do in a few months time too.

Reply:
```AddOnInstallConfirmation``` is the "Yes I want to install." It seems that that notification and the one after it are the same, meaning they have the same ```id```. I may try and see why the command on line 17 sometimes seems to also click the "OK" button after because sometimes that notification doesn't show.

Reply:
That's quite bad imho but okay, not sure that's changeable. Sounds like some upstream bug to make those notifications better identifiable?